docs-guides/source/multifunction-models.md

```diff
@@ -73,13 +69,15 @@ The weights for the base model will be shared.
 For an example, see the “Deploy machine learning and AI models on-device with Core ML” 
 [session video](https://developer.apple.com/videos/play/wwdc2024/10161/) in WWDC 2024
 to learn more about integrating multifunction models into your app. 
-It includes a text-to-image example in which a single 
+It includes a text-to-image example, in which a single 
 model with multiple adapters is used to generate images with different styles. 
 
 
 ## Combining models: toy example with LoRA adapters
 
-Lets define base model with two linear layers
+In this example, learn how to combine a base model and adapter model to create a merged multifunction Core ML model, load it, and call it to make predictions.
```

Comment:
Rather than "and call it to make predictions", I would probably say "and get predictions". Less words and also given our API, you're not really call it (i.e. the model), you're calling its' `predict` method. It's not `model(<args>)`, it's `model.predict(<args>)`.